packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -698,7 +698,9 @@
             "output": "Output",
             "includerow": "include column name row",
             "newline": "Newline",
-            "usestrings": "parse numerical values"
+            "usestrings": "parse numerical values",
+            "parse_empty_strings": "parse empty strings",
+            "parse_null_values": "parse null values"
```

Comment:
Likewise "include null values" - or is it null columns ?

Reply:
Done.